### DISCUSSION TITLE
Is there a way to reliably indicate which monitor full screen games open on without using Gamescope?

### DISCUSSION THREAD

**Discussion by winteriscariot** [labwc/labwc#2125] [Q&A]
My monitor layout:

![image](https://github.com/user-attachments/assets/573e9221-f7ae-42b3-9ae4-b072e8402532)

For some reason full screen games always open on the side panels (HDMI-A-2 or DP-2) instead of my main panel, DP-1

I can use gamescope in fullscreen mode and it will reliably open fullscreen on DP-1

however, I don't always want to use gamescope, for a number of reasons

is there a way, without using gamescope, that I can make sure full screen apps (in this case, games) always open on DP-1?

**Comment by johanmalm** [comment]
Are the games in question XWayland ones (e.g. steam based ones)? 

I'm just thinking that - as XWayland clients can position themselves - maybe the game just remembers where it was last positioned. Could you un-fullscreen and move it?
